pandas/core/groupby/groupby.py

```diff
@@ -923,17 +923,10 @@ def _python_agg_general(self, func, *args, **kwargs):
 
             try:
                 # if this function is invalid for this dtype, we will ignore it.
-                func(obj[:0])
+                result, counts = self.grouper.agg_series(obj, f)
```

Comment:
Any perf difference for numpy functions?

Reply:
i think this just means we dont fail-fast